megamek/src/megamek/server/Server.java

```diff
@@ -24823,6 +23938,7 @@ private void resolveShutdownCrashes() {
                 impactArmor = (te instanceof Mech)
                         && (te.getArmorType(hit.getLocation()) == EquipmentType.T_ARMOR_IMPACT_RESISTANT);
                 bar5 = te.getBARRating(hit.getLocation()) <= 5;
+                // TODO : Windchild the above is never used?
```

Comment:
The above is never used, so can it be removed or is it a bug?

Reply:
As near as I can tell with a method that's nearly 2k lines long, This is here to deal with the possibility of a different armor type when excess damage transfers to another location and was copied from the code that deals with the original hit. The only place it makes a difference is in the case of an acid warhead. I don't know the rules off the top of my head, but from what's in the code it does extra damage to some types of armor, including BAR <= 5, but the extra damage does not transfer. If that's correct, then it's not a bug and this line can be removed.